includes/handler/class-follow.php

```diff
@@ -72,23 +72,23 @@ public static function handle_follow( $activity ) {
 	/**
 	 * Send Accept response
 	 *
-	 * @param string                     $actor    The Actor URL
-	 * @param array                      $object   The Activity object
-	 * @param int                        $user_id  The ID of the WordPress User
-	 * @param Activitypub\Model\Follower $follower The Follower object
+	 * @param string                     $actor           The Actor URL
+	 * @param array                      $activity_object The Activity object
+	 * @param int                        $user_id         The ID of the WordPress User
+	 * @param Activitypub\Model\Follower $follower        The Follower object
 	 *
 	 * @return void
 	 */
-	public static function send_follow_response( $actor, $object, $user_id, $follower ) {
+	public static function send_follow_response( $actor, $activity_object, $user_id, $follower ) {
```

Comment:
because this is only an array, I would call it `$activity` to be consistent with the other code you changed above?!

Reply:
I would have liked to do that but there's a `$activity` variable set in line 108. Any preference?

Reply:
good point! then let's keep this one as is for now